doc/src/user_guide.rst

```diff
@@ -592,10 +617,11 @@ Example::
     quad-deg = 10
     quad-pts = gauss-legendre
 
-[solver-interfaces-tri]
-^^^^^^^^^^^^^^^^^^^^^^^
+[solver-interfaces-tri-{mg-*p*}]
```

Comment:
My fault, I got the syntax wrong, I did not realise we actually have a 'p'. So we could do:
solver-interfaces-tri{-mg-p*p*}
or following your other comment
solver-interfaces-tri{-mg-p*order*}